docs/ui/installation.rst

```diff
@@ -41,7 +41,7 @@ If you wish to run the full treeherder Vagrant project (service + UI), remember
 Running the unit tests
 ======================
 
-The unit tests run with Karma: http://karma-runner.github.io/0.8/config/configuration-file.html
+To locally validate your changes, the unit tests for the UI are run with Karma: http://karma-runner.github.io/0.8/config/configuration-file.html
```

Comment:
I think if someone doesn't know what unit tests are, we have bigger problems :-)
As such, I think the wording "The unit tests for the UI are run ..." is fine?
Happy to be convinced either way - but just think we could really do with simplifying our docs quite a bit - "less is more" - people stop reading after N words.